GVFS/FastFetch/Index.cs

```diff
@@ -213,14 +213,18 @@ private bool UpdateFileInformationFromDiskForFiles(MemoryMappedViewAccessor inde
                     addedOrEditedLocalFiles,
                     (localPath) =>
                     {
-                        string gitPath = FromWindowsFullPathToGitRelativePath(localPath, this.repoRoot);
+                        string gitPath = FromDotnetFullPathToGitRelativePath(localPath, this.repoRoot);
                         long offset;
                         if (this.indexEntryOffsets.TryGetValue(gitPath, out offset))
                         {
-                            if (this.TryUpdateEntryFromDisk(indexView, localPath, offset))
+                            if (NativeMethods.TryStatFileAndUpdateIndex(this.tracer, gitPath, indexView, offset))
                             {
                                 Interlocked.Increment(ref updatedEntriesFromDisk);
                             }
+                            else
+                            {
+                                this.tracer.RelatedError("Failed to update file information from disk for file {0}", gitPath);

```

Comment:
Not a big deal because these errors are only coming from FastFetch, but I've found it helpful to include the method name that's generating the error.

Example:

```
this.tracer.RelatedError($"{nameof(this.UpdateFileInformationFromDiskForFiles)}: Failed ...
```